Cannot start nuxt:  Cannot find module 'nuxt3'

### Environment

- Operating System: `Darwin`
- Node Version:     `v16.14.2`
- Nuxt Version:     `3.0.0-rc.1`
- Package Manager:  `npm@8.7.0`
- Builder:          `vite`
- User Config:      `runtimeConfig`, `modules`, `intlify`, `css`, `head`, `vite`
- Runtime Modules:  `@intlify/nuxt3@0.1.10`
- Build Modules:    `-`

### Reproduction

-

### Describe the bug

Commands `npx nuxi dev` and `npx nuxi build` throws error. Error can be seen in logs 

### Additional context

CONFIG
```ts
import { defineNuxtConfig } from "nuxt";

export default defineNuxtConfig({
  runtimeConfig: {
    DATABASE_URL: process.env.DATABASE_URL,
    CORAGEO_URL: process.env.CORAGEO_URL,
    CORAGEO_USER: process.env.CORAGEO_USER,
    CORAGEO_PASS: process.env.CORAGEO_PASS,
    CORAGEO_VERBOSE: process.env.CORAGEO_VERBOSE,
    ORGID: process.env.ORGID,
    UPLOAD_DIR: process.env.UPLOAD_DIR,
    API_URL: process.env.API_URL,
    EMAIL_VERIFICATION_EXPIRATION_DAYS: process.env.EMAIL_VERIFICATION_EXPIRATION_DAYS,
    PLATFORM: process.env.PLATFORM,
    SESSION_LIFESPAN_MINUTES: process.env.SESSION_LIFESPAN_MINUTES,
    RESET_PASS_EXPIRATION_DAYS: process.env.RESET_PASS_EXPIRATION_DAYS,
    public: {
      MAXSIZEMB: process.env.MAXSIZEMB,
      PAYMENT_URL: process.env.PAYMENT_URL,
      PAYMENT_KEY: process.env.PAYMENT_KEY,
    },
  },
  modules: ["@intlify/nuxt3"],
  intlify: {
    localeDir: "locales",
    vueI18n: {
      globalInjection: true,
      legacy: false,
      locale: "sk",
    },
  },
  css: ["@/assets/scss/app.scss"],
  head: {
    meta: [{ name: "viewport", content: "width=device-width, initial-scale=1" }],
  },
  vite: {
    css: {
      preprocessorOptions: {
        scss: {
          additionalData: "@import '@id-sk/frontend/idsk/all.scss';",
          quietDeps: true,
        },
      },
    },
  },
});

```
PACKAGE.JSON
```json
{
  "private": true,
  "scripts": {
    "dev": "nuxi dev",
    "build": "nuxi build",
    "preview": "nuxi preview",
    "start": "cross-env NUXT_HOST=0.0.0.0 node .output/server/index.mjs",
    "format": "prettier --config .prettierrc . --write"
  },
  "devDependencies": {
    "@intlify/nuxt3": "^0.1.10",
    "cross-env": "^7.0.3",
    "nuxt": "^3.0.0-rc.1",
    "prettier": "^2.6.2",
    "prisma": "^3.11.1"
  },
  "dependencies": {
    "@id-sk/frontend": "^2.9.0",
    "@prisma/client": "^3.11.1",
    "@vue-leaflet/vue-leaflet": "^0.6.1",
    "axios": "^0.26.1",
    "base64-img": "^1.0.4",
    "dayjs": "^1.11.0",
    "elmolis-corageo": "git+ssh://git@gitlab.elmolis.sk:Urbi/IntParkCoraGeo.git#devel",
    "hashids": "^2.2.10",
    "leaflet": "^1.7.1",
    "mustache": "^4.2.0",
    "sass": "^1.49.11",
    "sweetalert2": "^11.4.8"
  }
}

```

### Logs

```shell
ERROR  Cannot start nuxt:  Cannot find module 'nuxt3'                                                                                                                                  15:31:03
Require stack:
- /Users/enimo/Desktop/Projects/Elmolis/PortalPresov/index.js

  Require stack:
  - index.js
  at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
  at Function.resolve (node:internal/modules/cjs/helpers:108:19)
  at Function._resolve [as resolve] (node_modules/jiti/dist/jiti.js:1:192841)
  at resolveModule (node_modules/@nuxt/kit/dist/index.mjs:68:29)
  at importModule (node_modules/@nuxt/kit/dist/index.mjs:90:24)
  at loadNuxt (node_modules/@nuxt/kit/dist/index.mjs:798:43)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6733:23)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6777:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:46:20)
```

--- a/packages/nuxt3/package.json
+++ b/packages/nuxt3/package.json
@@ -43,7 +43,7 @@
     "@intlify/shared": "beta",
     "@intlify/vite-plugin-vue-i18n": "next",
     "@intlify/vue-i18n-loader": "next",
-    "@nuxt/kit": "npm:@nuxt/kit-edge@3.0.0-27277498.850ef69",
+    "@nuxt/kit": "latest",
     "pathe": "^0.2.0",
     "ufo": "^0.7.9",
     "debug": "^4.3.2",

